pkgs/applications/audio/argotlunar/default.nix

```diff
@@ -0,0 +1,48 @@
+ { pkgs } :
```

Comment:
This should explicitly list all the dependencies that you are currently accessing via `pkgs.` instead. This is what allows e.g. `mypkg.override` to work.

Example: `{ stdenv, lib, fetchFromGitHub, alsa-lib, freetype, curl, pkg-config, cmake, xorg }:`